src/app/servicesPage/services.js

```diff
@@ -32,50 +34,128 @@ angular.module( 'vyomo.servicesPage', [
+                                    var packagesJson = response.message.packages;
+                                    if(packagesJson.hasOwnProperty("all")){
+                                        var allPackages = packagesJson.all;
+                                        $scope.packages = _sortAccordingtoPrice(allPackages);
+                                        window.console.log($scope.packages);
+                                        // adding properties to package objs
+                                        $scope.packages.forEach(function(package){
+                                            addIsAddedToCartProperty(package);
+                                            // if package is in cart update cart price
+                                            updateCartPrice(package);
+                                        });
+
+                                    }
+
                                 }
-                                if (categories[i].label === "Menu") {
-                                    $scope.services = categories[i].services_by_group;
+
+                                if (response.message.hasOwnProperty('services')) {
+                                    var servicesJson = response.message.services;
+                                    if(servicesJson.hasOwnProperty("all")){
+                                        var allServices = servicesJson.all;
+                                        $scope.categories = allServices;
+                                        // adding properties to service objs
+                                        $scope.categories.forEach(function(category){
+                                            category.list.forEach(function(service){
+                                                addQuantityProperty(service);
+                                                // if service is in cart update cart price
+                                                updateCartPrice(service);
+                                            });
+                                        });
+                                        window.console.log($scope.categories);
+                                       
+                                    }
                                 }
-                            }
 
+                            }
                         }
                     }
+
                 });
                 $state.go('servicesPage.list');
             }
         };
+    
     }])
 
-    .controller( 'ServicePackageCtrl', function ServiceController() {
+    .controller( 'ServicePackageCtrl',['$scope', function ServiceController($scope) {
+
+        $scope.addOrRemovePackage = function(package){
+            // add package to cart if it is not
+            if(!package.isAddedToCart){
+                package.addToCart();
+            }
+            else{
+                package.removeFromCart();
```

Comment:
this can be written as 
`
package.isAddedToCart ? package.removeFromCart() : package.addToCart();
`